support/support.go

```diff
@@ -61,34 +63,47 @@ type Deps struct {
 }
 
 func DeploymentInfo(ctx context.Context, client *codersdk.Client, log slog.Logger) Deployment {
+	// Note: each goroutine assigns to a different struct field, hence no mutex.
 	var d Deployment
-
-	bi, err := client.BuildInfo(ctx)
-	if err != nil {
-		log.Error(ctx, "fetch build info", slog.Error(err))
-	} else {
+	eg, ctx := errgroup.WithContext(ctx)
```

Comment:
Just realized we only log errors, so we might be interested in partial results?

If so, might make sense to just use `eg := &errgroup.Group{}`. This way the other Go routines won't be interrupted when one runs into an error. (Sorry I didn't take this into account in my previous review.)